cpp/ql/src/semmle/code/cpp/ir/implementation/aliased_ssa/internal/SSASanity.qll

```diff
@@ -0,0 +1,2 @@
+private import SSAConstruction as SSA
+import SSA::SSASanity
```

Comment:
Why do we need this file? Can't we just write `import SSAConstruction::SSASanity` in `SSASanity.ql`?

Reply:
I think we could get rid of it, but it does fit the pattern we've used for `IRSanity.qll`, where each sanity query `.ql` file is just some metadata (which has to be specialized for the flavor of the IR) plus a single `import`.

Reply:
Okay. We can come back to this question as part of a later reform of the sanity code (moving it out of `Instruction.qll`).